contracts/integration-test/ZkEvmVerifierV1.spec.ts

```diff
@@ -65,7 +65,7 @@ describe("ZkEvmVerifierV1", async () => {
     // 0000000000000000000000000000000000000000000000000000000000000000 withdrawRoot
     // ee64d77c2f2e0b2c4ac952a0f54fdba4a217c42eb26a07b28de9fbc7b009acae batchDataHash
     // public input hash: 9ea439164727042e029464a40901e52800095c1ade301b63b4b7453880f5723e
-    expect(hexlify(publicInputHash)).to.eq("0x9ea439164727042e029464a40901e52800095c1ade301b63b4b7453880f5723e");
+    expect(hexlify(publicInputHash)).to.eq("0x31b430667bc9e8a8b7eda5e5c76f2250c64023f5f8e0689ac9f4e53f5362da66");
```

Comment:
comments should also be changed, or you can just remove it.

Reply:
Sorry, I just fixed it as CI prompt, please help review if correct, thanks.

Reply:
Update `public input hash` in commit [8291ab1](https://github.com/scroll-tech/scroll/pull/1049/commits/8291ab153081e0147dde11e91cd151e6e582bcb7).
Should I also update the above comment values as `withdrawRoot` and `batchDataHash`? Thanks.

Reply:
Yeah, you should.

Reply:
Any instruction about how to get the above values? Are they captured from `https://github.com/scroll-tech/test-traces/blob/674ad743beab04b57da369fa5958fb6824155bfe/erc20/1_transfer.json`? Thanks.

Or I should delete comments of these lines.

Reply:
I don't quite remember that, also maybe it only works for `v0.5.1`. That's why I suggest you remove it.

Reply:
I don't quite remember that, also maybe it only works for v0.5.1. That's why I suggest you remove it.

Reply:
Delete these lines of comments in commit [9cfd85e](https://github.com/scroll-tech/scroll/pull/1049/commits/9cfd85e648e0c67f23eebe657d16bdb8be98924a). Please help review if correct when you have time, thanks.